### PR TITLE
Update hero text.

### LINKED ISSUE #731: Modify homepage copy
The hero image of the new homepage has a text box that serves as an introduction to the site. The copy currently reads 'Detroit Home Connect is a place for you to find housing that you can afford based on your household needs and characteristics'. Safiya would like to remove 'and characteristics' from this copy so it becomes 'Detroit Home Connect is a place for you to find housing that you can afford based on your household needs.'
detroit_hero_image

### PR DESCRIPTION
**Pull request by plunkettgoogle**
## Issue

- Closes #731
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

Small update to the copy on the homepage.

Before:
Screenshot from 2021-11-04 15-53-30

After:
Screenshot from 2021-11-04 15-54-12

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [ ] This change is a refactor/addresses technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

Open the homepage.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes
- [ ] I have exported any new pieces in ui-components

### CODE REVIEW

**Comment by GabeBenjamin** [comment]
Can you add a before and after screenshot to the PR?

**Comment by plunkettgoogle** [comment]
Updated the PR comment. I'll do the same on my other PRs.

File: sites/public/page_content/locale_overrides/bn.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "welcome": {
     "title": "মধ্যে সাশ্রয়ী মূল্যের আবাসন খুঁজুন",
-    "heroText": "ডেট্রয়েট হোম কানেক্ট হল এমন একটি জায়গা যেখানে আপনি আপনার গৃহস্থালি চাহিদা এবং বৈশিষ্ট্যের উপর ভিত্তি করে বাসস্থান খুঁজে পেতে পারেন।"
+    "heroText": "ডেট্রয়েট হোম কানেক্ট হল এমন একটি জায়গা যেখানে আপনি আবাসন খুঁজে পেতে পারেন যা আপনি আপনার পরিবারের প্রয়োজনের ভিত্তিতে সামর্থ্য করতে পারেন।"
```
Comment (GabeBenjamin) [inline sites/public/page_content/locale_overrides/bn.json:4]: It looks like you specifically didn't translate "Detroit Home Connect" for the other languages, but you did here?

**Review by GabeBenjamin — APPROVED** [review]

File: sites/public/page_content/locale_overrides/bn.json (hunk 1 above)
Comment (plunkettgoogle) [reply sites/public/page_content/locale_overrides/bn.json]: I did my best guess based on Google translate here, hopefully it looks OK.

**Pull request merged**